### ISSUE TITLE
Window too small on HiDPI screens

### ISSUE THREAD

**Issue by forabi** [Browsers-software/browsers#160]
**Describe the bug**

**System:**
 - OS: [e.g. Ubuntu 23.10 with Gnome] Fedora 40 with Gnome 46 / Wayland / 4K Monitor
 - Browsers Version [e.g. 0.22.1] `⋮` -> About

**Comment by liias** [comment]
Hey, sorry for very late reply, but if there is a screenshot available with full desktop and browsers visible, that could help a b it.

**Comment by forabi** [comment]
@liias Here is a screenshot. My desktop has 175% scale. Environment: GNOME 46, Arch Linux (also Fedora). Wayland.
![image](https://github.com/user-attachments/assets/5f60595b-fbdb-454d-a193-3bc438e8947a)

**Comment by liias** [comment]
Hmm, I think the font really is simply a bit too small then, as it looks pretty correct from the screenshot actually. I'll play around with this one day.

**Comment by Victor239** [comment]
Yeah finding it way too small, and the window can't even be resized:

![Image](https://github.com/user-attachments/assets/784591d8-0eeb-438c-a354-75e797f08e1e)

**Comment by Shihabus-Sakib-Rad** [comment]
i have custom font size set up in KDE settings, but it doesn't use that
```
❯ cat ~/.config/gtk-4.0/settings.ini
[Settings]
gtk-application-prefer-dark-theme=true
gtk-cursor-blink=true
gtk-cursor-blink-time=1000
gtk-cursor-theme-name=breeze_cursors
gtk-cursor-theme-size=24
gtk-decoration-layout=icon:minimize,maximize,close
gtk-enable-animations=true
gtk-font-name=Noto Sans,  13
gtk-icon-theme-name=breeze-dark
gtk-primary-button-warps-slider=true
gtk-sound-theme-name=ocean
gtk-xft-dpi=98304
```
looks uncomfortably small  compared to junction. 
<img width="1920" height="1080" alt="Image" src="https://github.com/user-attachments/assets/ec55e2ed-06a5-48f3-990d-2116148eac9f" />
